src/raft.c

```diff
@@ -124,6 +124,10 @@ void RaftExecuteCommandArray(RedisModuleCtx *ctx,
     RedisRaftCtx *rr = &redis_raft;
     RedisModuleCallReply *reply;
 
+    if (rr->debug_delay_apply) {
```

Comment:
Don't we want to call it before each command exec?

Reply:
I want to call it before each entry/read request. If the request is MULTI, still I want to call it once. So, this is per entry slow down.